From 57185aeab4aed326c67f95f58fdfaa313adf5c34 Mon Sep 17 00:00:00 2001
From: George Dunlap <george.dunlap@citrix.com>
Date: Tue, 15 Mar 2016 14:21:24 +0000
Subject: [PATCH 02/17] Add blktap Kconfig

---
 drivers/block/Kconfig | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/drivers/block/Kconfig b/drivers/block/Kconfig
index 014a1cf..ff77cad 100644
--- a/drivers/block/Kconfig
+++ b/drivers/block/Kconfig
@@ -557,4 +557,16 @@ config BLK_DEV_RSXX
 	  To compile this driver as a module, choose M here: the
 	  module will be called rsxx.
 
+config XEN_BLKDEV_TAP
+       tristate "Block-device tap backend driver"
+       depends on XEN_BACKEND
+       default XEN_BACKEND
+       help
+         The block tap driver is an alternative to the block back driver
+         and allows VM block requests to be redirected to userspace through
+         a device interface.  The tap allows user-space development of
+         high-performance block backends, where disk images may be implemented
+         as files, in memory, or on other hosts across the network.  This
+         driver can safely coexist with the existing blockback driver.
+
 endif # BLK_DEV
-- 
2.1.4

