From 33eef0e06b57a733af84cad0062c7afc705b91c9 Mon Sep 17 00:00:00 2001
From: George Dunlap <george.dunlap@citrix.com>
Date: Tue, 15 Mar 2016 14:21:59 +0000
Subject: [PATCH] Wire in missing bnx2 firmware

---
 firmware/Makefile | 68 ++++++++++++++++++++++++++++++++++++++++++++++++-------
 1 file changed, 60 insertions(+), 8 deletions(-)

diff --git a/firmware/Makefile b/firmware/Makefile
index e297e1b..cd31e33 100644
--- a/firmware/Makefile
+++ b/firmware/Makefile
@@ -31,14 +31,66 @@ fw-shipped-$(CONFIG_ADAPTEC_STARFIRE) += adaptec/starfire_rx.bin \
 					 adaptec/starfire_tx.bin
 fw-shipped-$(CONFIG_ATARI_DSP56K) += dsp56k/bootstrap.bin
 fw-shipped-$(CONFIG_ATM_AMBASSADOR) += atmsar11.fw
-fw-shipped-$(CONFIG_BNX2X) += bnx2x/bnx2x-e1-6.2.9.0.fw \
-			      bnx2x/bnx2x-e1h-6.2.9.0.fw \
-			      bnx2x/bnx2x-e2-6.2.9.0.fw
-fw-shipped-$(CONFIG_BNX2) += bnx2/bnx2-mips-09-6.2.1a.fw \
-			     bnx2/bnx2-rv2p-09-6.0.17.fw \
-			     bnx2/bnx2-rv2p-09ax-6.0.17.fw \
-			     bnx2/bnx2-mips-06-6.2.1.fw \
-			     bnx2/bnx2-rv2p-06-6.0.15.fw
+fw-shipped-$(CONFIG_BNX2X) += bnx2x/bnx2x-e1-6.0.34.0.fw \
+                              bnx2x/bnx2x-e1-6.2.5.0.fw \
+                              bnx2x/bnx2x-e1-6.2.9.0.fw \
+                              bnx2x/bnx2x-e1-7.0.20.0.fw \
+                              bnx2x/bnx2x-e1-7.0.23.0.fw \
+                              bnx2x/bnx2x-e1-7.0.29.0.fw \
+                              bnx2x/bnx2x-e1-7.10.51.0.fw \
+                              bnx2x/bnx2x-e1-7.2.16.0.fw \
+                              bnx2x/bnx2x-e1-7.2.51.0.fw \
+                              bnx2x/bnx2x-e1-7.8.17.0.fw \
+                              bnx2x/bnx2x-e1-7.8.19.0.fw \
+                              bnx2x/bnx2x-e1-7.8.2.0.fw \
+                              bnx2x/bnx2x-e1h-6.0.34.0.fw \
+                              bnx2x/bnx2x-e1h-6.2.5.0.fw \
+                              bnx2x/bnx2x-e1h-6.2.9.0.fw \
+                              bnx2x/bnx2x-e1h-7.0.20.0.fw \
+                              bnx2x/bnx2x-e1h-7.0.23.0.fw \
+                              bnx2x/bnx2x-e1h-7.0.29.0.fw \
+                              bnx2x/bnx2x-e1h-7.10.51.0.fw \
+                              bnx2x/bnx2x-e1h-7.2.16.0.fw \
+                              bnx2x/bnx2x-e1h-7.2.51.0.fw \
+                              bnx2x/bnx2x-e1h-7.8.17.0.fw \
+                              bnx2x/bnx2x-e1h-7.8.19.0.fw \
+                              bnx2x/bnx2x-e1h-7.8.2.0.fw \
+                              bnx2x/bnx2x-e2-6.0.34.0.fw \
+                              bnx2x/bnx2x-e2-6.2.5.0.fw \
+                              bnx2x/bnx2x-e2-6.2.9.0.fw \
+                              bnx2x/bnx2x-e2-7.0.20.0.fw \
+                              bnx2x/bnx2x-e2-7.0.23.0.fw \
+                              bnx2x/bnx2x-e2-7.0.29.0.fw \
+                              bnx2x/bnx2x-e2-7.10.51.0.fw \
+                              bnx2x/bnx2x-e2-7.2.16.0.fw \
+                              bnx2x/bnx2x-e2-7.2.51.0.fw \
+                              bnx2x/bnx2x-e2-7.8.17.0.fw \
+                              bnx2x/bnx2x-e2-7.8.19.0.fw \
+                              bnx2x/bnx2x-e2-7.8.2.0.fw 
+fw-shipped-$(CONFIG_BNX2) +=  bnx2/bnx2-mips-06-4.6.16.fw \
+                              bnx2/bnx2-mips-06-5.0.0.j3.fw \
+                              bnx2/bnx2-mips-06-5.0.0.j6.fw \
+                              bnx2/bnx2-mips-06-6.0.15.fw \
+                              bnx2/bnx2-mips-06-6.2.1.fw \
+                              bnx2/bnx2-mips-06-6.2.3.fw \
+                              bnx2/bnx2-mips-09-4.6.17.fw \
+                              bnx2/bnx2-mips-09-5.0.0.j15.fw \
+                              bnx2/bnx2-mips-09-5.0.0.j3.fw \
+                              bnx2/bnx2-mips-09-5.0.0.j9.fw \
+                              bnx2/bnx2-mips-09-6.0.17.fw \
+                              bnx2/bnx2-mips-09-6.2.1.fw \
+                              bnx2/bnx2-mips-09-6.2.1a.fw \
+                              bnx2/bnx2-mips-09-6.2.1b.fw \
+                              bnx2/bnx2-rv2p-06-4.6.16.fw \
+                              bnx2/bnx2-rv2p-06-5.0.0.j3.fw \
+                              bnx2/bnx2-rv2p-06-6.0.15.fw \
+                              bnx2/bnx2-rv2p-09-4.6.15.fw \
+                              bnx2/bnx2-rv2p-09-5.0.0.j10.fw \
+                              bnx2/bnx2-rv2p-09-5.0.0.j3.fw \
+                              bnx2/bnx2-rv2p-09-6.0.17.fw \
+                              bnx2/bnx2-rv2p-09ax-5.0.0.j10.fw \
+                              bnx2/bnx2-rv2p-09ax-5.0.0.j3.fw \
+                              bnx2/bnx2-rv2p-09ax-6.0.17.fw 
 fw-shipped-$(CONFIG_CASSINI) += sun/cassini.bin
 fw-shipped-$(CONFIG_CHELSIO_T3) += cxgb3/t3b_psram-1.1.0.bin \
 				   cxgb3/t3c_psram-1.1.0.bin \
-- 
2.1.4

